docs/BuildFromSource.md

```diff
@@ -108,6 +108,10 @@ NodeJS installs the Node package manager (npm) by default. This repo depends on
 npm install -g yarn
 ```
 
+#### [tar](http://gnuwin32.sourceforge.net/packages/gtar.htm) on Windows
+
+Building the repo requires tar to be installed (Win10 comes with tar already installed). You can find installation excutables of tar at <http://gnuwin32.sourceforge.net/packages/gtar.htm>. You should also add the installation directory to your PATH variable.
```

Comment:
Because `C:\windows\system32\tar.exe` usually exists on Windows systems, I'd prefer this paragraph was specific to the systems (maybe Windows7 or early Windows Server versions❔) where `tar` is missing by default.

Reply:
Good news is https://docs.microsoft.com/en-us/virtualization/community/team-blog/2017/20171219-tar-and-curl-come-to-windows was released years ago. Bad news is it's not clear the addition was completed in all Windows versions.

Reply:
> 
> 
> Because `C:\windows\system32\tar.exe` usually exists on Windows systems, I'd prefer this paragraph was specific to the systems (maybe Windows7 or early Windows Server versionsgrey_question) where `tar` is missing by default.

Yes, I try to build the repo on a Win7 machine

Reply:
Alright, how about
```suggestion
Building the repo requires tar to be installed. First, check whether `tar.exe` is already in your path i.e. execute `tar -help` (Win10 comes with tar already installed). Then, assuming you have `git` installed, you might add `C:\Program Files\Git\usr\bin\` to your path to pick up the `tar.exe` that ships with `git`. Finally, you can find installation executables of tar at <http://gnuwin32.sourceforge.net/packages/gtar.htm>; download that and add the installation directory to your PATH variable.
```

Reply:
I agree.